datadog-agent.yaml

```diff
@@ -138,6 +140,19 @@ pipeline:
 
       invoke -e deps
 
+  - name: "Enable JMX-based check"
+    runs: |
+      # get the latest version of the JMX
+      # https://github.com/DataDog/datadog-agent/blob/85aa2772bea8dc39ffc0d49f05c3288a67d4ba18/docs/dev/checks/jmxfetch.md#stable-jmxfetch
+      version=$(jq -r '.["release-a7"].JMXFETCH_VERSION' release.json)
+      curl -sSLO https://oss.sonatype.org/service/local/repositories/releases/content/com/datadoghq/jmxfetch/$version/jmxfetch-$version-jar-with-dependencies.jar
```

Comment:
Please do not do this.

Instead create new package, called jmxfetch and build it from source with maven https://github.com/DataDog/jmxfetch?tab=readme-ov-file#building-from-source as that's the only way we can then rebuild that jar; bump jar dependencies and mitigate vulnerabilities.